bridge/discord/discord.go

```diff
@@ -287,11 +287,30 @@ func (b *Bdiscord) handleEventBotUser(msg *config.Message, channelID string) (st
 		return msg.ID, err
 	}
 
+	if msg.ParentID != "" && msg.ParentID != "msg-parent-not-found" {
+		m := discordgo.MessageSend{
+			Reference: &discordgo.MessageReference{
+				MessageID: msg.ParentID,
+				ChannelID: channelID,
+				GuildID:   b.guildID,
+			},
```

Comment:
If the message reference is the only difference, can we just use ChannelMessageSendComplex every time?

Reply:
Good point